### PR TITLE
The previous file should be the absolute path, not the string used to import it.

### PR DESCRIPTION
**Pull request by chriseppstein**
node-sass 3.4 had a breaking public API change. This patch fixes the path that is passed as the previous file path when resolving imports. In node-sass 3.4 this value changed so that the import uri was passed. The issue here is that any code needed to resolve one import against another quickly falls apart if the resolved path returned by the importer is not what is passed back to the importer as the current sass file.

Basically, I think the updates for node-sass 3.4 to use libsass 3.3 mistakenly used the wrong value.

### CODE REVIEW

**Comment by mgreter** [comment]
Related to https://github.com/sass/node-sass/issues/1219 (see my comments in there, which basically comes to the same conclusion).

**Comment by xzyfer** [comment]
Thanks @chriseppstein this will land in v3.4.2 in the next couple days.

**Comment by vlad** [comment]
Thanks for fixing this, hope this lands soon!
